stock-usecase/src/main/java/org/dhis2/android/rtsm/ui/home/screens/components/MainContent.kt

```diff
@@ -120,7 +122,14 @@ fun MainContent(
                     .background(color = Color.White, shape = RoundedCornerShape(30.dp))
                     .weight(1 - (weightValue + weightValueArrow))
                     .alignBy(FirstBaseline)
-                    .align(alignment = Alignment.CenterVertically),
+                    .align(alignment = Alignment.CenterVertically)
+                    .onFocusEvent {
```

Comment:
what is the difference between this and:
`.onFocusChanged { focusState ->
            if (focusState.isFocused) {`